src/components/FormValidator.js

```diff
@@ -78,7 +78,13 @@ export default class FormValidator {
   enableValidation() {
     this._formEl.addEventListener("submit", (evt) => {
       evt.preventDefault();
-      this.disableButton();
+      this.submitForm()
+        .then(() => {
+          this.disableButton();
+        })
+        .catch((err) => {
+          console.error("Form submission error:", err);
```

Comment:
* When I submit any form, there is an error https://snipboard.io/45e7SO.jpg 


Please, delete `this.submitForm()` from `FormValidator.js`.


You should keep the old code here, but without `this.disableButton();`


```js

    this._formEl.addEventListener("submit", (evt) => {
      evt.preventDefault();
    });

```